### PR TITLE
:sparkles: by default messages are adressed to the room when a user i…

### PR DESCRIPTION
**Pull request by antoineeudes**

### CODE REVIEW

File: src/components/organisms/Chatbox/Chatbox.tsx [hunk 1]
```diff
@@ -91,31 +127,49 @@ const Chatbox: React.FunctionComponent<PropsType> = ({
...
                   </Dropdown.Menu>
                 </Dropdown>
               </div>
             )}
 
             <ChatForm
-              type={isPrivate ? "private" : chatboxMessageType}
-              discussionPartner={discussionPartner}
+              type={
+                isPrivate || privateRecipient ? "private" : chatboxMessageType
```
Comment (mathieud) [inline src/components/organisms/Chatbox/Chatbox.tsx:168]: Do we have 2 types of private messages?

File: src/components/organisms/Chatbox/Chatbox.tsx [hunk 2]
```diff
@@ -65,11 +77,35 @@ const Chatbox: React.FunctionComponent<PropsType> = ({
...
 
+  let privateRecipient;
+  const lastChat = chatsToDisplay && chatsToDisplay[0];
+  if (
+    !isPrivate &&
+    !hasUserSelectedRecipient &&
+    lastChat &&
+    lastChat.type === "private"
+  ) {
+    if (lastChat?.to === user.uid) {
```
Comment (mathieud) [inline src/components/organisms/Chatbox/Chatbox.tsx:93]: I don't know how active the chat is, but it shouldn't change who you're sending it to while you're writing something.

File: src/components/organisms/Chatbox/Chatbox.tsx [hunk 3]
```diff
@@ -50,26 +56,64 @@ const Chatbox: React.FunctionComponent<PropsType> = ({
...
   }
 
+  const changeChatboxMessageType = (type: string) => {
+    setIsRecipientChangeBlocked(true);
+    setChatboxMessageType(type);
+    setPrivateRecipient(undefined);
+  };
+
+  useEffect(() => {
+    console.log(isRecipientChangeBlocked);
```
Comment (mathieud) [inline src/components/organisms/Chatbox/Chatbox.tsx:93]: remove this

File: src/components/organisms/Chatbox/Chatbox.tsx (hunk 2 above)
Comment (thecadams) [reply src/components/organisms/Chatbox/Chatbox.tsx]: Seems like a pretty reasonable point, lets raise this point to chat with Ed and I in TR

Comment (thecadams) [reply src/components/organisms/Chatbox/Chatbox.tsx]: Ah just noticed this got addressed as I kept reading, please disregard! Great work!

**Pull request merged**